DeepCat.js

```diff
@@ -647,9 +647,17 @@
 		}
 	}
 
-	$( function() {
-		deepCatMain();
-	} );
+	if( mw.loader.getState( 'ext.gadget.DeepCat' )  !==  'ready' ) {
+		mw.loader.state( 'ext.gadget.DeepCat', 'ready' );
+		mw.loader.using(
+			[ 'mediawiki.api',
```

Comment:
can we have this one in its own line too?

BTW: I don't see mw.api (not mw.api.messages) being used in a code. Do you (need to) add it because of mw.api.messages?